# HG changeset patch
# User Rik <rik@octave.org>
# Date 1350230494 25200
#      Sun Oct 14 09:01:34 2012 -0700
# Node ID dee69050bb0202cff390419cca93aba21b591ec9
# Parent  3f1b306e2ba94ae02d7b9e4a438bb4cc96fb473a
rgbplot.m: Tweak docstring.

* rgbplot.m: Tweak docstring.

diff --git a/scripts/image/rgbplot.m b/scripts/image/rgbplot.m
--- a/scripts/image/rgbplot.m
+++ b/scripts/image/rgbplot.m
@@ -13,17 +13,17 @@
 ## General Public License for more details.
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn  {Function File} {} rgbplot (@var{cmap})
-## @deftypefnx {Function File} {@var{h} =} rgbplot (@dots{})
+## @deftypefnx {Function File} {@var{h} =} rgbplot (@var{cmap})
 ## Plot the components of a colormap.
 ##
 ## The first column is plotted in red, the second column in green, and
 ## the third column in blue.  The values are between 0 and 1 and represent
 ## the intensity of the RGB components in the given indexed color.
 ##
 ## The optional return value @var{h} is a graphics handle to the created plot.
 ##
